app/controllers/api/v1/jobs.rb

```diff
@@ -0,0 +1,30 @@
+module API
+  module V1
+    class Jobs < Base
+
+        resources :jobs do
+          desc 'return all of the jobs'
+          get do
+            Job.all
```

Comment:
this one is a little bit risky in a case when you will have a lot of jobs. Limit or pagination should help you here